exporter/sentryexporter/interfaces.go

```diff
@@ -15,12 +15,17 @@
 package sentryexporter
 
 import (
+	"encoding/json"
 	"time"
 
 	"github.com/getsentry/sentry-go"
 )
 
-// Tags describes a Sentry Tag.
+// SentryEvent aliases the sentry Event type.
+// Needed to Marshal the transactions into JSON properly.
+type SentryEvent sentry.Event
```

Comment:
```suggestion
type sentryEvent sentry.Event
```

@AbhiPrasad could you check if an unexported type would work here? I think it should.

This type definition should be temporary until we upstream changes into the SDK, but I don't want to accidentally export symbols into the public API unnecessarily.